docs/docs/index.md

```diff
@@ -103,6 +103,64 @@ q.enqueue('my_package.my_module.my_func', 3, 4)
 ```
 
 
+### CLI Enqueueing
+
+If you prefer enqueueing jobs via the command line interface or do not use python
+you can use this.
+
+
+#### Usage:
+```bash
+rq enqueue [OPTIONS] FUNCTION [ARGUMENTS]
+```
+
+#### Options:
+* `-q, --queue [value]`      The name of the queue.
+* `--timeout [value]`        Specifies the maximum runtime of the job before it is
+                               interrupted and marked as failed.
+* `--result-ttl [value]`     Specifies how long successful jobs and their results
+                               are kept.
+* `--ttl [value]`            Specifies the maximum queued time of the job before
+                               it is discarded.
+* `--failure-ttl [value]`    Specifies how long failed jobs are kept.
+* `--description [value]`    Additional description of the job
+* `--depends-on [value]`     Specifies another job id that must complete before this
+                               job will be queued.
+* `--job-id [value]`         The id of this job
+* `--at-front`               Will place the job at the front of the queue, instead
+                               of the end
+* `--retry-max [value]`      Maximum number of retries
+* `--retry-interval [value]` Interval between retries in seconds
+* `--schedule-in [value]`    Delay until the function is enqueued (e.g. 10s, 5m, 2d).
+* `--schedule-at [value]`    Schedule job to be enqueued at a certain time formatted
+                               in ISO 8601 without timezone (e.g. 2021-05-27T21:45:00).
+* `--quiet`                  Only logs errors.
+
+#### Function:
+There are two options:
+* Execute a function: dot-separated string of package, module and function (Just like
+    passing a string to `queue.enqueue()`).
+* Execute a python file: dot-separated pathname of the file. Because it is technically
+    an import `__name__ == '__main__'` will not work.
+
+#### Arguments:
+
+| | plain text | json | [literal-eval](https://docs.python.org/3/library/ast.html#ast.literal_eval) |
+|-|-|-|-|
+| keyword | `[key]=[value]` | `[key]:=[value]` | `[key]#=[value]` |
```

Comment:
Can we give a few examples of how we can enqueue things with JSON or Python objects?

Reply:
Where do you mean? In the docs?

Reply:
Yes.